File: app/src/main/java/com/leoart/koreanphrasebook/ui/chapters/models/Category.java
package com.leoart.koreanphrasebook.ui.chapters.models;

/**
 * Created by bogdan on 11/5/16.
 */
public class Category {
    private String id;
    private String name;
    private String chapter;

    public Category() {

    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public Category(String name) {
        this.name = name;
    }

    public String getCategory() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public String getChapter() {
        return chapter;
    }

    public void setChapter(String chapter) {
        this.chapter = chapter;
    }
}

File: app/src/main/java/com/leoart/koreanphrasebook/ui/chapters/models/Category.kt
package com.leoart.koreanphrasebook.ui.chapters.models

import com.google.firebase.database.Exclude
import java.util.*

/**
 * Created by bogdan on 11/5/16.
 */
class Category(val id: String, val name: HashMap<String, String>) {

    @Exclude
    fun toMap(): HashMap<String, Any> {
        val map = HashMap<String, Any>()
        // map.putAll(name.toMap())
        // map.put("id", translation)
        //map.put("chapter", chapter)
        // map.put("index", index)
        return map
    }
}
